js/main.js

```diff
@@ -104,3 +143,11 @@ function makeColumnHalf() {
   col.setAttribute('class', 'column-half');
   return col;
 }
+
+function findEntryId(entryId) {
+  for (var i = 0; i < data.entries.length; i++) {
```

Comment:
Generally speaking it's best practice to pass in variables you'll use as arguments into the function -- so it'd be a good idea for this to have two parameters and you pass in `data.entries` as one of them.

I say this because when functions use variables from the global scope directly like this, it can become more difficult to understand where those are coming from.